pig/src/main/java/org/jboss/pnc/bacon/pig/impl/addons/quarkus/VertxArtifactFinder.java

```diff
@@ -0,0 +1,146 @@
+package org.jboss.pnc.bacon.pig.impl.addons.quarkus;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.jboss.pnc.bacon.pig.impl.PigContext;
+import org.jboss.pnc.bacon.pig.impl.addons.AddOn;
+import org.jboss.pnc.bacon.pig.impl.config.PigConfiguration;
+import org.jboss.pnc.bacon.pig.impl.pnc.PncBuild;
+import org.jboss.pnc.bacon.pig.impl.utils.FileUtils;
+import org.jboss.pnc.bacon.pig.impl.utils.MavenRepositoryUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import static org.jboss.pnc.bacon.pig.impl.utils.FileUtils.mkTempDir;
+
+/**
+ * @author Saumya Singh, singhsaumyas150@gmail.com <br>
+ *         Date: 25/03/2021
+ */
+
```

Comment:
Hi @SaumyaSingh1 !

I know we're not consistent enough with this, but could you please add a description of the Add-on here that we could then also put in our documentation? Thank you! :)

Reply:
Otherwise LGTM! :rocket:

Reply:
Sure thing @thescouser89 :) 
I should have added a description earlier.

>we could then also put in our documentation

If you could direct me to the link/doc where I have to add details of this Add-on, I would love to do that :)

Reply:
Thank you!

The specific section is: https://project-ncl.github.io/bacon/guide/pig.html#add-ons

Which is from: https://github.com/project-ncl/bacon/blob/gh-pages/guide/pig.md#add-ons

It's the same repository, but in the `gh-pages` branch. So you can create your own branch from `gh-pages`, then do the changes to the `pig.md` file, and create a PR targetting the `gh-pages` branch.

As you can see, we're doing a really poor job at maintaining the add-on section. Thank you for your help on improving our documentation! :heart:

Reply:
Sure thing Dustin, I will make a PR for it :) 
I really liked the neat documentation of Bacon https://project-ncl.github.io/bacon/.

Reply:
That's all thanks to @rnc ! He's the magician behind it :)